platform-hub-web/web.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:6.9.1
+
+WORKDIR /app
+COPY . .
+
+RUN npm install -g yarn@0.27.5
+RUN git config --global url."https://".insteadOf git://
+RUN yarn install
```

Comment:
This may actually require `gulp` CLI tool.

Reply:
The `gulp` not found issue has been resolved.
* Was due to `docker-compose.yml` initial volume not allow the `node_modules` folder to exist. See [stackoverflow post](https://stackoverflow.com/questions/30043872/docker-compose-node-modules-not-present-in-a-volume-after-npm-install-succeeds#:~:text=This%20happens%20because%20you%20have,the%20dependencies%20are%20installed%20there.).